pkg/dockerregistry/server/auth.go

```diff
@@ -110,7 +110,7 @@ func (ac *authChallenge) Error() string {
 }
 
 // SetHeaders sets the basic challenge header on the response.
-func (ac *authChallenge) SetHeaders(w http.ResponseWriter) {
+func (ac *authChallenge) SetHeaders(req *http.Request, w http.ResponseWriter) {
```

Comment:
This was made because an interface somewhere else has changed?

Reply:
Yes, see vendor/github.com/docker/distribution/registry/auth/auth.go.